server/lomas_server/admin_database/admin_database.py

```diff
@@ -419,24 +420,24 @@ def prepare_save_query(
         """
         to_archive = {
             "user_name": user_name,
-            "dataset_name": query_json.dataset_name,
-            "dp_librairy": model_input_to_lib(query_json),
-            "client_input": query_json.model_dump(),
-            "response": response,
+            "dataset_name": query.dataset_name,
+            "dp_librairy": model_input_to_lib(query),
+            "client_input": query.model_dump(),
+            "response": response.model_dump(),
             "timestamp": time.time(),
-        }
+        }  # TODO use model for that one too.
```

Comment:
ah yes, i see the TODO (i commented in the client), it would be nice to add a ticket and the ticket number next to the #TODO :)

Reply:
Good point, I'll create the ticket and update the comment